source/_components/sensor.thermoworks_smoke.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: "Thermoworks Smoke Sensor"
+description: "Pulls temperature data for a Thermoworks Smoke Thermometer connected with Smoke Gateway."
+date: 2018-08-22 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: thermoworks.png
+ha_category: Sensor
+ha_release: 0.78.0
+ha_iot_class: "Remote Polling"
+---
+
+The `thermoworks_smoke` sensor platform pulls data for your [Thermoworks Smoke Thermometer](https://www.thermoworks.com/Smoke).
+This requires a [Smoke WiFi Gateway](https://www.thermoworks.com/Smoke-Gateway) with an internet connection.
+
+You will need to have previously registered your smoke to your account via the mobile app and provide 
+the email and password you used to in the configuration for this sensor in order to connect and pull your data.
+
+## {% linkable_title Configuration %}
+
+To add the sensors to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: thermoworks_smoke
+    email: "your email here"
+    password: !secret thermoworks_pass
+    monitored_conditions:
+    - probe1
+    - probe2
+    - probe1_min
+    - probe1_max
+    - probe2_min
+    - probe2_max
+```
+
+{% configuration %}
+email:
+  description: The email address with the device registered in the thermoworks smoke mobile app.
+  required: true
+  type: string
+password:
+  description: The password registered in the thermoworks smoke mobile app.
+  required: true
+  type: string
+monitored_conditions:
+  description: The sensors to add. Default is `probe1` and `probe2`.
+  required: false
+  type: list
+exclude:
+  description: Device serial numbers to ignore.
+  required: false
+  type: list
+{% endconfiguration %}
+
+## {% linkable_title Examples %}
+
+In this section you find some real life examples of how to use this sensor.
```

Comment:
- missing comma after section `this section, you`
- `real life` -> `real-life`

Reply:
@nhorvath Thanks for pointing that out, made a PR to fix those other docs: https://github.com/home-assistant/home-assistant.io/pull/6384